makefile

```diff
@@ -1,7 +1,7 @@
 # start project configuration
 name := cedar
 buildDir := build
-packages := $(name) rest rest-data rest-model units operations cost model depgraph perf rpc rpc-internal
+packages := $(name) rest rest-data rest-model units operations cost model depgraph perf rpc rpc-internal depot
```

Comment:
maybe we name this certdepot just so it's different than the depot package in certstrap